app/uk/gov/hmrc/tai/model/tai/TaxYear.scala

```diff
@@ -49,7 +49,7 @@ case class TaxYear(year: Int) extends Ordered[TaxYear] {
     (currentDate.isEqual(TaxYear().start) || currentDate.isAfter(TaxYear().start)) &&
       (currentDate.isBefore(TaxYear().end) || currentDate.isEqual(TaxYear().end))
 
-  @deprecated("Does not check if date is post tax year, use withinTaxYear instead")
+  @deprecated("Does not check if date is post tax year, use withinTaxYear instead", since = "0.299.0")
   def fallsInThisTaxYear(currentDate: LocalDate): Boolean = {
```

Comment:
Is this still used?
I noticed the change in `app/uk/gov/hmrc/tai/calculators/EstimatedPayCalculator.scala` was refactored to use `withinTaxYear`, so could this method be phased out entirely?

Reply:
This can be removed as it's not used anywhere.